tests/tests_complexity.py

```diff
@@ -1,18 +1,23 @@
+import math
```

Comment:
Do we need that? i don't see any call to `math`

Reply:
Thank you for going over it in detail.

There are two calls to `math` in line 596 and line 598, which did not get changed and do not show in the commit message.

`isort` moved the input to the top, before it was in front of the test with the math calls.